src/foremast/templates/configs/pipeline.json.j2

```diff
@@ -20,5 +20,11 @@
         "handler": "main",
         "vpc_enabled": false
     },
-    "pipeline_files": []
+    "pipeline_files": [],
+    "chaos_monkey": {
+      "enabled": false,
+      "mean_time": 5,
+      "min_time": 3,
+      "exceptions": ["prod", "prods", "prodp"]
```

Comment:
`prod*` accounts should not be in the open source templates. These should be in our private templates